scipy/stats/_continuous_distns.py

```diff
@@ -9972,11 +9972,71 @@ class vonmises_gen(rv_continuous):
     `vonmises_line` is the same distribution, defined on :math:`[-\pi, \pi]`
     on the real line. This is a regular (i.e. non-circular) distribution.
 
-    `vonmises` and `vonmises_line` take ``kappa`` as a shape parameter.
+    Note about distribution parameters: `vonmises` and `vonmises_line` take
+    ``kappa`` as a shape parameter and the ``scale`` parameter does not have
+    any effect.
 
-    %(after_notes)s
+    Examples
+    --------
+    Import the necessary modules.
 
-    %(example)s
+    >>> import numpy as np
+    >>> import matplotlib.pyplot as plt
+    >>> from scipy.stats import vonmises
+
+    Define distribution parameters: the circular mean $\mu$ has to be passed
+    via the ``loc`` (location) parameter and the concentration parameter
+    $\kappa$ as ``kappa``.
+
+    >>> loc = 0.5 * np.pi
+    >>> kappa = 1
+
+    Compute the probability density at ``x=0`` via the ``pdf`` method.
+
+    >>> vonmises.pdf(loc, kappa, 0)
+    0.12570826359722018
+
+    Verify that the percentile function ``ppf`` inverts the cumulative
+    distribution function ``cdf`` up to floating point accuracy.
+
+    >>> x = 1
+    >>> cdf_value = vonmises.cdf(loc=loc, kappa=kappa, x=x)
+    >>> ppf_value = vonmises.ppf(cdf_value, loc=loc, kappa=kappa)
+    >>> x, cdf_value, ppf_value
+    (1, 0.31489339900904967, 1.0000000000000004)
+
+    Draw 1000 random variates by calling the ``rvs`` method.
+
+    >>> number_of_samples = 1000
+    >>> samples = vonmises(loc=loc, kappa=kappa).rvs(number_of_samples)
+
+    Plot the von Mises density on a cartesian and polar grid to emphasize
+    that is is a circular distribution.
+
+    >>> fig = plt.figure(figsize=(8, 6))
+    >>> left = plt.subplot(121)
```

Comment:
Consider using an equal aspect. I think that would render better as it would not "squeeze" the figure.

Reply:
Not sure I fully understand all the wonders matplotlib can do here.

Setting `aspect=True` does not have any effect on the polar plot but squeezes the cartesian one almost invisible.

![image](https://user-images.githubusercontent.com/40656107/236331271-71585ba7-7004-4fac-baed-bbed8cc30431.png)


<details><summary>Plot generation</summary>
<p>
import numpy as np
import matplotlib.pyplot as plt
from scipy.stats import vonmises

x = np.linspace(-np.pi, np.pi, 500)

fig = plt.figure(figsize=(8, 6))
left = plt.subplot(121, aspect="equal")
right = plt.subplot(122, projection='polar', aspect="equal")

# distribution parameters
loc = 0.5 * np.pi  # mu, specifies circular mean
kappa = 1  # concentration parameter kappa
number_of_samples = 1000

# draw random variates
samples = vonmises(loc=loc, kappa=kappa).rvs(number_of_samples)
ticks = [0, 0.15, 0.3]
vonmises_pdf = vonmises.pdf(loc, kappa, x)
pdf_plot = left.plot(x, vonmises_pdf)
left.set_yticks(ticks)
number_of_bins = int(np.sqrt(number_of_samples))
left.hist(samples, density=True, bins=number_of_bins)
left.set_title("Cartesian plot")
left.grid(True)

right.plot(x, vonmises_pdf, label="PDF")
right.set_yticks(ticks)
right.hist(samples, density=True, bins=number_of_bins, label="Histogram")
right.set_title("Polar plot", pad=65)
right.legend(bbox_to_anchor=(0.3, 1.35))

fig.suptitle(r"Von Mises distribution: $\mu=\frac{\pi}{2},\ \kappa=1$")
plt.show()


</p>
</details> `

Reply:
It looks like this did about what we'd expect. To satisfy the intent of the plot, please make the plot on the left a little wider and less tall than what it is now:
![image](https://user-images.githubusercontent.com/6570539/236352610-6a2777a2-b8fd-4c82-a2a0-74db37c4fa03.png)

Reply:
Thanks, yes aspect is maybe not working here. As Matt is suggesting my comment is about making the 2 plot roughly the same size and not look "too compressed".